L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryV9Imp2.cc

```diff
@@ -348,6 +438,16 @@ HGCalTriggerGeometryBase::geom_ordered_set HGCalTriggerGeometryV9Imp2::getOrdere
       }
     }
   }
+  // HFNose
+  else if (det == DetId::Forward && DetId(module_det_id).subdetId() == ForwardSubdetector::HFNose) {
+    HFNoseDetId module_nose_id(module_id);
+    HFNoseDetIdToModule hfn;
+    std::vector<HFNoseTriggerDetId> ids = hfn.getTriggerDetIds(module_nose_id);
+    for (unsigned int tdi = 0; tdi < ids.size(); tdi++) {
```

Comment:
Would be more readable with a range-based loop

Reply:
done